docs/dev/setup.sh

```diff
@@ -1,34 +1,60 @@
 #!/bin/sh
 
-stream_backend="$1"
-
-if [ -z "$stream_backend" ] || (echo "stan jetstream" | grep -v -q "$stream_backend"); then
-  stream_backend="stan"
-fi
-echo "Using $stream_backend"
-
+OSTYPE=$(uname -s)
 COMPOSE_CMD='docker-compose'
+CLUSTER_NAME="demo-a"
+COMPOSE_FILE="./docs/dev/armada-compose.yaml"
 
-DCLIENT_VERSION=$(docker version -f '{{.Client.Version}}')
-if echo $DCLIENT_VERSION | grep -q '^2' ; then
+docker compose > /dev/null 2>&1
+if [ $? -eq 0 ] ; then
   COMPOSE_CMD='docker compose'
 fi
 
-kind create cluster --name demo-a --config ./docs/dev/kind.yaml
-
-OSTYPE=$(uname -s)
 if [ $OSTYPE == "Linux" ]; then
```

Comment:
You comment out pulsar in the compose file so I think the profile flag does nothing.

Reply:
I was undecided if Pulsar bits should be left in or not - I'll rip all the Pulsar stuff out. If someone really wants it, it can be added later.

Reply:
Removed in commit d7573ebd.